### DISCUSSION TITLE
LFX Mentorship '24 Term 2 Jun - Aug: Open for Project Proposals

### DISCUSSION THREAD

**Discussion by nate-double-u** [cncf/mentoring#1205] [Announcements]
Hello everyone!

The LFX Mentorship 2024 Term 2 Jun - Aug is open now open for project ideas:
https://github.com/cncf/mentoring/blob/main/programs/lfx-mentorship/2024/02-Jun-Aug

Project submission and application timeline:
* project proposal period: April 8 - May 8
* mentee applications open: May 13 - May 28
* application review/admission decisions/HR paperwork: May 29 - June 11

We're looking forward to seeing all the project ideas you're interested in working on!

### Timeline

| activity | date |
| --- | --- |   
| project proposals | Monday April 8 - Wed May 8, 2024, 5:00 PM PDT |
| mentee applications open | Monday May 13 - Tues May 28, 5:00 PM PDT |
| application review/admission decisions | Wed May 29 - Tues June 11, 5:00 PM PDT |
| selection notifications | Wed June 12, 5:00 PM PDT |
| Mentorship program begins with the initial work assignments | Monday June 17 (Week 1) | 
| Midterm mentee evaluations and first stipend payments | Wednesday July 24 (Week 6) |
| Final mentee evaluations and mentee feedback/blog submission due, second and final stipend payment approvals | Wed Aug 28, 5:00 PM PST (Week 12) |
| Last day of term | Friday Aug 30 |

**Comment by saswatamcode** [comment]
Could the dates here be updated to reflect the ones in https://github.com/cncf/mentoring/tree/main/programs/lfx-mentorship/2024/02-Jun-Aug @nate-double-u? Got confused by the deadline 🙂

**Reply by nate-double-u** [reply]
Yes we can 😅

Sorry about that, will update it shortly, we _just_ extended the deadlines yesterday to sync better with the GSoC program.

**Comment by nate-double-u** [comment]
Just a note everyone, we've made an update to the Term 2 schedule: Project proposals are welcome for another couple weeks 🎉
This pushes most things in the schedule back a week or two.

**Comment by Himanshu-kiran** [comment]
When will the cncf organization will release the all projects ideas ?

**Comment by pankaj87072** [comment]
hey @nate-double-u is the process is same as gsoc for applying in LFX mentorship program

**Comment by Himanshu-kiran** [comment]
Hello @pankaj87072 , core structure of program is same as both are open source program but yes there are some differences in application process.

**Reply by pankaj87072** [reply]
ohk

**Reply by pankaj87072** [reply]
and we have to do some contribution first right before submitting application

**Reply by Himanshu-kiran** [reply]
There is nothing like compulsion. But yes if you could feel like solving any issue, raising pull request before submission then obviously it's great.
Go ahead.

**Reply by pankaj87072** [reply]
thanks for the help sir i will do my best

**Comment by Forchapeatl** [comment]
Hi , has the selection results for The LFX Mentorship 2024 Term 2 Jun - Aug been announced yet ? It is past the due date.

**Reply by nate-double-u** [reply]
Accepted candidates have been notified, and we're working on notifying everyone else. I appreciate your patience, it's a manual process.

**Comment by Himanshu-kiran** [comment]
@Forchapeatl  Yep, was thinking same. 
It has been not annouced yet.
